src/python/nnfusion/executor.py

```diff
@@ -218,4 +227,17 @@ def feed_data(self, inputs, outputs, strict=True):
 
     def feed_pointers(self, signature, params):
         self.kernel_entry.argtypes = signature
-        self.kernel_entry(*params)
\ No newline at end of file
+        self.kernel_entry(*params)
+
+    def _maybe_reserve_mem(self, device):
+        if not hasattr(self.libnnf, "get_workspace_size"):
```

Comment:
This `if` is unnecessary?

Reply:
Yes, I was going to fix this at the same time I fix the unit bug.